spec/abstractops.html

```diff
@@ -196,51 +196,54 @@ <h1>ToShowOffsetOption ( _normalizedOptions_ )</h1>
     <h1>
       ToTemporalRoundingIncrement (
         _normalizedOptions_: an Object,
-        _dividend_: an integer or *undefined*,
+      ): either a normal completion containing a Number, or an abrupt completion
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>
+        It extracts the value of the property named *"roundingIncrement"* from _normalizedOptions_, makes sure it is a finite Number greater than or equal to 1, and returns that value.
+        It performs no further validation.
+      </dd>
+    </dl>
+    <emu-alg>
+      1. Let _increment_ be ? GetOption(_normalizedOptions_, *"roundingIncrement"*, *"number"*, *undefined*, *1*<sub>𝔽</sub>).
+      1. If _increment_ is not finite, throw a *RangeError* exception.
+      1. If _increment_ &lt; *1*<sub>𝔽</sub>, throw a *RangeError* exception.
+      1. Return _increment_.
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-validatetemporalroundingincrement" type="abstract operation">
+    <h1>
+      ValidateTemporalRoundingIncrement (
+        _increment_: a Number,
+        _dividend_: a positive integer,
         _inclusive_: a Boolean,
-      )
+      ): either a normal completion containing an integer, or an abrupt completion
     </h1>
     <dl class="header">
       <dt>description</dt>
       <dd>
-        It extracts the value of the property named *"roundingIncrement"* from _normalizedOptions_, makes sure it is a valid value for the option, and returns that value rounded to an integer.
-        If _dividend_ is *undefined*, any non-zero positive value is valid.
-        Otherwise, the value must not be more than _dividend_ (or must be less than _dividend_, if _inclusive_ is *false*) and must divide evenly into _dividend_.
+        It performs further validation of a value _increment_ returned from ToTemporalRoundingIncrement.
+        It checks whether _increment_ divides evenly into _dividend_, and throws an exception if not.
+        If _inclusive_ is *true*, _increment_ is allowed to be equal to _dividend_, otherwise it must be less.
```

Comment:
```suggestion
        It truncates an _increment_ from ToTemporalRoundingIncrement to an integer
        and returns the result if it evenly divides _dividend_ (otherwise throwing a *RangeError*).
        _dividend_ must be divided into more than one part unless _inclusive_ is *true*.
```

But things would probably be a bit cleaner overall if the rounding were part of ToTemporalRoundingIncrement rather than ValidateTemporalRoundingIncrement (presuming the change in my other comment).

Reply:
That change is made in #2437.